tasks.py

```diff
@@ -164,6 +166,8 @@ def setup(ctx):
         migrate(ctx)
         print("* Creating fake data.")
         manage(ctx, "load_fake_data")
+        print("* Sync locales")
```

Comment:
hm, this is shared between new-db and new-env, if we're adding more shared functionality, let's make this its own `def` to capture the code starting at "applying database migrations" and then have both invoke tasks call that.